src/app/home/info/console-pipeline-panel.component.sass

```
@@ -0,0 +1,101 @@
+:host
+  display: block
```

Stylelint (17.4.0)

[error] Unknown word display (CssSyntaxError)

(CssSyntaxError)